lib/v2/gdut/oa_news.js

```diff
@@ -0,0 +1,210 @@
+                        const newSty = this.attribs.style
+                            .split(';')
+                            .filter((s) => {
+                                const styBlocklist = ['color:rgb(0,0,0)', 'color:black', 'background:rgb(255,255,255)', 'background:white', 'text-align:left', 'text-align:justify', 'font-style:normal', 'font-weight:normal'];
+                                const styPrefixBlocklist = [
+                                    'font-family',
+                                    'font-size',
+                                    'background',
+                                    'text-autospace',
+                                    'text-transform',
+                                    'letter-spacing',
+                                    'line-height',
+                                    'padding',
+                                    'margin',
+                                    'text-justify',
+                                    'word-break',
+                                    'vertical-align',
+                                    'mso-',
+                                    '-ms-',
+                                ];
+                                const sty = s.trim();
+                                if (styBlocklist.includes(sty.replace(/\s+/g, ''))) {
+                                    return false;
+                                }
+                                for (const prefix of styPrefixBlocklist) {
+                                    if (sty.startsWith(prefix)) {
+                                        return false;
+                                    }
+                                }
+                                return true;
+                            })
+                            .join(';');
+                        if (newSty) {
+                            this.attribs.style = newSty;
+                        } else {
+                            delete this.attribs.style;
+                        }
+                    }
+                    if (this.attribs.class) {
+                        if (this.attribs.class.trim().startsWith('Mso')) {
+                            delete this.attribs.class;
+                        }
+                    }
+                    if (this.attribs.lang) {
+                        delete this.attribs.lang;
+                    }
+                    if (this.tagName === 'font' || this.tagName === 'o:p') {
+                        $(this).replaceWith(this.childNodes);
+                    }
+                    if (this.tagName === 'span' && !this.attribs.style) {
+                        $(this).replaceWith(this.childNodes);
+                    }
+                });
+                node.find('span').each(function () {
+                    if (this.childNodes.length === 0) {
+                        $(this).remove();
+                    }
+                });
+
+                // 防止太快被Ban
+                await wait(500);
```

Comment:
`Promise.all()` will run each function of `data` item in `articles` in parallel which means this `wait` will only block each return but not stop them.
You can use `tiny-async-pool` to limit concurrent requests.